schemas/bedrock-profile-http.js

```diff
@@ -0,0 +1,185 @@
+  type: 'string'
+};
+
+// this should match query objects with an account in them
+const accountQuery = {
+  title: 'Accout Query',
+  type: 'object',
+  required: ['account'],
+  // FIXME set this to false once we have
+  // decided on all the properties.
+  additionalProperties: true,
+  properties: {account}
+};
+
+const profileAgent = {
+  title: 'Profile Agent',
+  type: 'object',
+  required: ['profile'],
+  // FIXME set this to false once we have
+  // decided on all the properties.
+  additionalProperties: true,
+  properties: {
+    account,
+    profile,
+    token: {
+      title: 'Token',
+      type: 'string'
+    }
+  }
+};
+
+const profileAgents = {
+  title: 'Profile Agents',
+  type: 'object',
+  // FIXME set this to false once we have
+  // decided on all the properties.
+  additionalProperties: true,
+  properties: {
+    account,
+    profile
+  }
+};
+
+const delegateCapability = {
+  title: 'Delegate Capability',
+  type: 'object',
+  required: ['invoker', 'account'],
+  // FIXME set this to false if we
+  // have decided on all the properties
+  additionalProperties: true,
+  properties: {
+    account,
+    invoker: {
+      anyOf: [{
+        type: 'string'
+      }, {
+        type: 'array',
+        minItems: 1,
+        items: {type: 'string'}
+      }]
+    }
+  }
+};
+
+const zcap = {
+  title: 'zcap',
+  type: 'object',
+  // FIXME set this to false once we have
+  // decided on all the properties.
+  additionalProperties: true,
+  properties: {
+    id: {
+      title: 'id',
+      type: 'string'
+    },
+    allowedAction: {
+      anyOf: [{
+        type: 'string'
+      }, {
+        type: 'array',
+        minItems: 1,
+        items: {type: 'string'}
+      }]
+    },
+    caveat: {
+      title: 'Caveat',
+      type: 'object'
+    },
+    '@context': {
+      title: '@context',
+      anyOf: [{
+        type: 'string'
+      }, {
+        type: 'array',
+        minItems: 1,
+        items: {anyOf: [{type: 'string'}, {type: 'object'}]}
+      }, {
+        type: 'object'
+      }]
+    },
+    controller: {
+      title: 'controller',
+      type: 'string'
+    },
+    delegator: {
+      anyOf: [{
+        type: 'string'
+      }, {
+        type: 'array',
+        minItems: 1,
+        items: {type: 'string'}
+      }]
+    },
+    invoker: {
+      anyOf: [{
+        type: 'string'
+      }, {
+        type: 'array',
+        minItems: 1,
+        items: {type: 'string'}
+      }]
+    },
+    invocationTarget: {
+      title: 'Invocation Target',
+      anyOf: [{
+        type: 'string'
+      }, {
+        type: 'object',
+        properties: {
+          id: {
+            title: 'Invocation Target Id',
+            type: 'string'
+          },
+          type: {
+            title: 'Invocation Target Type',
+            type: 'string'
+          }
+        }
+      }]
+    },
+    parentCapability: {
+      title: 'Parent Capability',
+      anyOf: [{type: 'string'}, {type: 'object'}]
```

Comment:
Do we ever use objects here? If not, let's restrict this to just strings.

Reply:
That might cause issues with other projects. I think restricting it to just a string is to strong.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-profile-http/pull/10/commits/77038a0960cdf291be06ec761367bea3217b5a11